src/plugins/dashboard/public/dashboard_app/top_nav/controls_toolbar_button/controls_toolbar_button.tsx

```diff
@@ -17,7 +17,12 @@ import { AddDataControlButton } from './add_data_control_button';
 import { AddTimeSliderControlButton } from './add_time_slider_control_button';
 import { EditControlGroupButton } from './edit_control_group_button';
 
-export function ControlsToolbarButton({ controlGroup }: { controlGroup: ControlGroupContainer }) {
+interface Props {
+  controlGroup: ControlGroupContainer;
+  isDisabled?: boolean;
+}
+
+export const ControlsToolbarButton: FC<Props> = ({ controlGroup, isDisabled }) => {
```

Comment:
nit: why the use of FC here? Using FC like this [implicitly adds](https://medium.com/raccoons-group/why-you-probably-shouldnt-use-react-fc-to-type-your-react-components-37ca1243dd13) the `children` prop, and this toolbar button doesn't actually take children. It's also a little cleaner to just declare the Props type inline instead of naming it.